package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medic",
-  "version": "3.11.1",
+  "version": "3.11.2",
```

Comment:
Terrible work. Please rethink the design and refactor this.

Reply:
:trollface:

just kidding of course. approving...

Reply:
Oh noooooooooo